### PR TITLE
Add Alerts

### LINKED ISSUE #215: readd alerts
**Describe the solution you'd like**
We should re add support for alert boxes like
```
:::success
Yes :tada:
:::

:::info
This is a message :mega:
:::

:::warning
Watch out :zap:
:::

:::danger
Oh No! :fire:
:::
```

grafik

**Additional context**
This should be rather straight forward, since we can just uses the `Alert` component of react-bootstrap.

### PR DESCRIPTION
**Pull request by mrdrogdrog**
### Component/Part
The markdown renderer

### Description
This PR adds the `alerts` feature to the markdown renderer

### Steps

- [x] added implementation
- [x] added / updated tests
- [x] added / updated documentation
- [x] extended changelog

### Related Issue(s)
Fixes #215

### CODE REVIEW

File: src/components/editor/markdown-preview/markdown-preview.tsx [hunk 1]
```diff
@@ -41,7 +42,12 @@ const MarkdownPreview: React.FC<MarkdownPreviewProps> = ({ content }) => {
     md.use(markdownItRegex, replaceYouTubeLink)
     md.use(markdownItRegex, replaceVimeoLink)
     md.use(markdownItRegex, replaceGistLink)
-    md.use(MarkdownItParserDebugger)
+    md.use(markdownItContainer, 'success', { render: createRenderContainer('success') })
+    md.use(markdownItContainer, 'danger', { render: createRenderContainer('danger') })
+    md.use(markdownItContainer, 'info', { render: createRenderContainer('info') })
+    md.use(markdownItContainer, 'warning', { render: createRenderContainer('warning') })
```
Comment (ErikMichelson) [inline src/components/editor/markdown-preview/markdown-preview.tsx:48]: Would it be more clean to use a list and loop over it here?

File: src/components/editor/markdown-preview/markdown-preview.tsx [hunk 2]
```diff
@@ -41,7 +42,12 @@ const MarkdownPreview: React.FC<MarkdownPreviewProps> = ({ content }) => {
...
+
+    // md.use(MarkdownItParserDebugger)
```
Comment (ErikMichelson) [inline src/components/editor/markdown-preview/markdown-preview.tsx:50]: Is disabling the parser-debugger plugin intended?

Comment (mrdrogdrog) [reply src/components/editor/markdown-preview/markdown-preview.tsx:50]: no. reenabled

File: src/components/editor/markdown-preview/markdown-preview.tsx (hunk 1 above)
Comment (mrdrogdrog) [reply src/components/editor/markdown-preview/markdown-preview.tsx]: done

**Pull request merged**